package.json

```diff
@@ -118,6 +117,7 @@
     ]
   },
   "dependencies": {
-    "deepmerge": "^4.0.0"
+    "deepmerge": "^4.0.0",
+    "vue-clipboard2": "^0.3.1"
```

Comment:
should be a dev dependency - the above as well

Reply:
phoenix will never be installed via yarn - no dependency has to be taken into consideration - all deps are bundled during dev time -> only dev dependencies for phoenix

Reply:
Fixed